src/telemetry/index.ts

```diff
@@ -262,8 +263,8 @@ function sendTelemetryEventInternal<P extends IEventNamePropertyMapping, E exten
 
         reporter.sendTelemetryEvent(eventNameSent, customProperties, measures);
     }
-
-    traceVerbose(
+    traceWithLevel(
```

Comment:
Oh moved this to an even higher trace level. Telemetry spew was bothering me too.